website/versioned_docs/version-0.9.5/reference/cyber.md

```diff
@@ -18,65 +18,65 @@ sidebar_label: CyberML
    (i.e., it returns a sample from the complement set).
 
 ## feature engineering: [indexers.py](https://github.com/microsoft/SynapseML/blob/master/core/src/main/python/synapse/ml/cyber/feature/indexers.py)
-1. [IdIndexer](https://mmlspark.blob.core.windows.net/docs/0.9.4/pyspark/synapse.ml.cyber.feature.html#synapse.ml.cyber.feature.indexers.IdIndexer)
+1. [IdIndexer](https://mmlspark.blob.core.windows.net/docs/0.9.5/pyspark/synapse.ml.cyber.feature.html#synapse.ml.cyber.feature.indexers.IdIndexer)
```

Comment:
maybe we should just have a "latest" tag here in the future?  Changing all of these links is error prone on every version update.

Reply:
This is a good idea and we will investigate how to do this

Reply:
We'll just update the `next` version, and all previous staged version won't be changed then.